errors.go

```diff
@@ -53,18 +53,20 @@ var (
 	ERR_MNS_QPS_LIMIT_EXCEEDED           = errors.TN(ALI_MNS_ERR_NS, 134, ali_MNS_ERR_TEMPSTR)
 	ERR_MNS_UNKNOWN_CODE                 = errors.TN(ALI_MNS_ERR_NS, 135, ali_MNS_ERR_TEMPSTR)
 
-	ERR_MNS_TOPIC_NAME_LENGTH_ERROR      = errors.TN(ALI_MNS_ERR_NS, 200, ali_MNS_ERR_TEMPSTR)
-	ERR_MNS_SUBSRIPTION_NAME_LENGTH_ERROR= errors.TN(ALI_MNS_ERR_NS, 201, ali_MNS_ERR_TEMPSTR)
-	ERR_MNS_TOPIC_NOT_EXIST              = errors.TN(ALI_MNS_ERR_NS, 202, ali_MNS_ERR_TEMPSTR)
-	ERR_MNS_TOPIC_ALREADY_EXIST          = errors.TN(ALI_MNS_ERR_NS, 203, ali_MNS_ERR_TEMPSTR)
-	ERR_MNS_INVALID_TOPIC_NAME           = errors.TN(ALI_MNS_ERR_NS, 204, ali_MNS_ERR_TEMPSTR)
-	ERR_MNS_INVALID_SUBSCRIPTION_NAME    = errors.TN(ALI_MNS_ERR_NS, 205, ali_MNS_ERR_TEMPSTR)
-	ERR_MNS_SUBSCRIPTION_ALREADY_EXIST   = errors.TN(ALI_MNS_ERR_NS, 206, ali_MNS_ERR_TEMPSTR)
-	ERR_MNS_INVALID_ENDPOINT             = errors.TN(ALI_MNS_ERR_NS, 207, ali_MNS_ERR_TEMPSTR)
-	ERR_MNS_SUBSCRIBER_NOT_EXIST         = errors.TN(ALI_MNS_ERR_NS, 211, ali_MNS_ERR_TEMPSTR)
+	ERR_MNS_TOPIC_NAME_LENGTH_ERROR        = errors.TN(ALI_MNS_ERR_NS, 200, ali_MNS_ERR_TEMPSTR)
+	ERR_MNS_SUBSCRIPTION_NAME_LENGTH_ERROR = errors.TN(ALI_MNS_ERR_NS, 201, ali_MNS_ERR_TEMPSTR)
```

Comment:
这样改完，用户的代码可能会不兼容？

Reply:
这个主要是我们用吧，我们目前没有用到这个值，而且用户使用的话，一般是直接使用这个值SubscriptionNameLengthError， map 中的key值，这个是没有改变的

Reply:
大概就加两行在下面

```
// ERR_MNS_SUBSRIPTION_NAME_LENGTH_ERROR: discarded because of  typo
ERR_MNS_SUBSRIPTION_NAME_LENGTH_ERROR = ERR_MNS_SUBSCRIPTION_NAME_LENGTH_ERROR
```

Reply:
good service